apps/remote-storage-server/src/entities/entities.service.ts

```diff
@@ -30,7 +30,7 @@ export class EntitiesService {
 
     const entityKey = this.getEntityKey(actor, key)
 
-    return await this.dataServiceFactory.getService().set(entityKey, value)
+    return await this.dataServiceFactory.getService().set(entityKey, value,new Date().getTime())
```

Comment:
Here and everywhere else, please enable `prettier` and use the config in the repo to fix code style issues.